cfpb/consumerfinance.gov
# Adding "Other" category for Notices

This PR adds the category "Other" to the Notice and Opportunity for Comment `category` list.


## How to test this PR

1. localhost
2. Create a new Document Detail page
3. Check the available category listings, under "Notice and Opportunity for Comment"
